src/stamp.cr

```diff
@@ -0,0 +1,81 @@
+class Hashcash::Stamp
+  STAMP_VERSION = 1
```

Comment:
perhaps it's worth using a string here? to runtime save a conversion to string for every parse, up to you